### PR TITLE
Dataflow strategy: use traces from the build, auto-focus, no fork (#1346).

### PR DESCRIPTION
**Pull request by Dor1s**
Instead of `-collect_data_flow=<DATAFLOW_BINARY> -fork=1`, use `-data_flow_trace=<DATAFLOW_TRACE_DIR> -focus_function=auto` and intentionally do not use the fork mode (at least for now).

### CODE REVIEW

**Comment by Dor1s** [comment]
Marty, could you also review this one, and I'll deploy all recent changes at once then?

File: src/python/bot/fuzzers/libfuzzer.py [hunk 1]
```diff
@@ -1599,8 +1605,8 @@ def pick_strategies(strategy_pool, fuzzer_path, corpus_directory,
   is_ephemeral = environment.is_ephemeral()
 
   if (not is_fuchsia and not is_android and not is_ephemeral and
-      (use_dataflow_tracing or
-       strategy_pool.do_strategy(strategy.FORK_STRATEGY))):
+      not use_dataflow_tracing and
```
Comment (inferno-chromium) [inline src/python/bot/fuzzers/libfuzzer.py]: So, you don't want fork mode to be used in dataflow tracing anymore ?

Comment (Dor1s) [reply src/python/bot/fuzzers/libfuzzer.py]: yes

Comment (mbarbella-chromium) [reply src/python/bot/fuzzers/libfuzzer.py]: Could you add a comment explaining why they should be mutually exclusive here? Do they actually need to be mutually exclusive or could we still occasionally pick both?

Comment (Dor1s) [reply src/python/bot/fuzzers/libfuzzer.py]: Done, also fixed a stale comment (that somehow went too above in the code), thanks!

The strategies are not mutually exclusive, but in the fork mode the log is pretty poor, so I want to get readable logs from the DFT runs for now. They include coverage information, focus function selected, and some other DFT-specific stats.

**Pull request merged**